ansible/roles/csm.gpg_keys/tasks/main.yml

```diff
@@ -23,7 +23,7 @@
 #
 # Tasks for the csm.gpg_keys role
 - name: Fetch Public GPG Keys from the K8S secret
-  no_log: true
+# no_log: true
```

Comment:
Do not check in commented lines of code -- just remove them if they are no longer needed.

```suggestion
```